### PR TITLE
added error in conditioning

### PR DESCRIPTION
**Pull request by MatheMax**

### CODE REVIEW

File: R/ContinuousPrior.R [hunk 1]
```diff
@@ -93,11 +93,22 @@ setMethod("condition", signature("ContinuousPrior", "numeric"),
...
+
+        interval[1] <- max(interval[1], dist@support[1])
+        interval[2] <- min(interval[2], dist@support[2])
+        if (diff(interval) < 0)
+            stop("resulting interval is empty")
         # compute new normalizing constant
         z <- stats::integrate(dist@pdf, interval[1], interval[2], abs.tol = .00001)$value
+        new_pdf <- function(theta) {
+            res <- 0
+            if(interval[1] <= theta && theta <= interval[2])
```
[the remark this answers is not in the document]
Comment (MatheMax) [reply R/ContinuousPrior.R]: I think this is vectorized, but I added a test for this

[the remark this answers is not in the document]
Comment (kkmann) [reply R/ContinuousPrior.R]: @MatheMax your test case does not catch that problem + I am 99% sure it is not - plz read up on the difference between if/ifelse again ;) and esp. the behaviour of if () for vectors...

File: R/ContinuousPrior.R [hunk 2]
```diff
@@ -93,11 +93,22 @@ setMethod("condition", signature("ContinuousPrior", "numeric"),
...
             stop("interval[2] must be larger or equal to interval[1]")
+
+        interval[1] <- max(interval[1], dist@support[1])
+        interval[2] <- min(interval[2], dist@support[2])
+        if (diff(interval) < 0)
+            stop("resulting interval is empty")
         # compute new normalizing constant
         z <- stats::integrate(dist@pdf, interval[1], interval[2], abs.tol = .00001)$value
+        new_pdf <- function(theta) {
+            res <- 0
```
[the remark this answers is not in the document]
Comment (kkmann) [reply R/ContinuousPrior.R]: i guess this is resolved ...

**Comment by kkmann** [comment]
lol, now I broke it ;)

**Comment by kkmann** [comment]
fucking comma, srsly?

**Comment by kkmann** [comment]
okay, it was a bit more involved than anticipated ... 

the problem is that you used if instead of ifelse and then tried to fix the fact that if complains about not-length-one arguments by using && instead of & ;)

again, ifelse docs + https://stat.ethz.ch/R-manual/R-devel/library/base/html/Logic.html should clarify the problem...

**Pull request merged**